axelor-account/src/main/java/com/axelor/apps/account/service/invoice/attributes/InvoiceLineAttrsServiceImpl.java

```diff
@@ -58,4 +73,8 @@ public void setCompanyInTaxTotalScale(
         currencyScaleServiceAccount.getCompanyScale(invoice),
         attrsMap);
   }
+
+  protected String computeField(String field, String prefix) {
+    return prefix != null ? prefix.concat(".").concat(field) : field;
```

Comment:
Please use `String.format` and add the `.` to `prefix` value when calling method so that you only have `String.format("%s%s", prefix, field);`. And maybe use `""` instead of `null` when there's no prefix (see with Arthur)